code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -310,15 +310,14 @@
 /obj/item/ammo_casing/caseless/foam_dart/update_icon()
 	..()
 	if(modified)
+		desc = "This foam dart doesn't look too safe"
 		icon_state = "foamdart_empty"
-		desc = "Its nerf or nothing! ... Although, this one doesn't look too safe."
```

Comment:
Don't change the description or order unless there's an explicit reason to. In this case, there isn't.